### PR TITLE
PR:i51 : Updating the functions to reflect accurate info

### PR DESCRIPTION
**Pull request by kanjoe24**
Improving the log messages in ut_kvp.c.
Also, changed one of the debug messages to error log message to better suit the requirement.

### CODE REVIEW

**Comment by kanjoe24** [comment]
Release test script results:

```
 ./release-test-script-ut-control.sh -t feature/gh51-fix-relative-path
==========================================================
RESULTS for ubuntu 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
Openssl static lib does not exist. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for VM-SYNC 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
build/linux/openssl/lib/libssl.a exists. PASS 
host-tools/CMake-3.30.0/build/bin/cmake exists. PASS 
==========================================================
==========================================================
RESULTS for dunfell_arm 
On the branch feature/gh51-fix-relative-path. PASS
build/arm/curl/lib/libcurl.a exists. PASS
build/arm/openssl/lib/libssl.a exists. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for dunfell_linux 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
Openssl static lib does not exist. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for kirkstone_arm 
On the branch feature/gh51-fix-relative-path. PASS
build/arm/curl/lib/libcurl.a exists. PASS
build/arm/openssl/lib/libssl.a exists. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for kirkstone_linux 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
Openssl static lib does not exist. PASS 
CMake host binary does not exist. PASS 
==========================================================

```

**Review by Ulrond — CHANGES_REQUESTED** [review]
ah sorry forgot to push this one out.

File: src/ut_kvp.c [hunk 1]
```diff
@@ -476,7 +476,7 @@ bool ut_kvp_fieldPresent( ut_kvp_instance_t *pInstance, const char *pszKey)
     node = fy_node_by_path(root, zKey, -1, FYNWF_DONT_FOLLOW);
     if ( node == NULL )
     {
-        UT_LOG_ERROR("node not found: UT_KVP_STATUS_KEY_NOT_FOUND");
+        UT_LOG_ERROR("node not found for key = %s : UT_KVP_STATUS_KEY_NOT_FOUND", pszKey);
```
[the remark this answers is not in the document]
Comment (kanjoe24) [reply src/ut_kvp.c:479]: sorry, why will that not be an error. We are returning false if don't find the node for the said key. I didn't understand...

File: src/ut_kvp.c [hunk 2]
```diff
@@ -476,7 +476,8 @@ bool ut_kvp_fieldPresent( ut_kvp_instance_t *pInstance, const char *pszKey)
     node = fy_node_by_path(root, zKey, -1, FYNWF_DONT_FOLLOW);
     if ( node == NULL )
     {
-        UT_LOG_ERROR("node not found: UT_KVP_STATUS_KEY_NOT_FOUND");
+        //UT_LOG_DEBUG("node not found for key = [%s] : UT_KVP_STATUS_KEY_NOT_FOUND", pszKey);
```
Comment (kanjoe24) [inline src/ut_kvp.c:479]: Upgrade logging module to have levels of logging and also instances

**Review by Ulrond — APPROVED** [review]
looks good

File: src/ut_kvp.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (kanjoe24) [reply src/ut_kvp.c]: done

**Comment by kanjoe24** [comment]
test Script results:
```
./release-test-script-ut-control.sh -t feature/gh51-fix-relative-path
==========================================================
RESULTS for kirkstone_arm 
On the branch feature/gh51-fix-relative-path. PASS
build/arm/curl/lib/libcurl.a exists. PASS
build/arm/openssl/lib/libssl.a exists. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for ubuntu 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
Openssl static lib does not exist. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for VM-SYNC 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
build/linux/openssl/lib/libssl.a exists. PASS 
host-tools/CMake-3.30.0/build/bin/cmake exists. PASS 
==========================================================
==========================================================
RESULTS for dunfell_arm 
On the branch feature/gh51-fix-relative-path. PASS
build/arm/curl/lib/libcurl.a exists. PASS
build/arm/openssl/lib/libssl.a exists. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for dunfell_linux 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
Openssl static lib does not exist. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for kirkstone_arm 
On the branch feature/gh51-fix-relative-path. PASS
build/arm/curl/lib/libcurl.a exists. PASS
build/arm/openssl/lib/libssl.a exists. PASS 
CMake host binary does not exist. PASS 
==========================================================
==========================================================
RESULTS for kirkstone_linux 
On the branch feature/gh51-fix-relative-path. PASS
build/linux/curl/lib/libcurl.a exists. PASS
Openssl static lib does not exist. PASS 
CMake host binary does not exist. PASS 
==========================================================
```

**Pull request merged**